pkgs/data/icons/candy-icons/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenvNoCC
+, fetchzip
+, gtk3
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "candy-icons";
+  version = "0.6.8";
+
+  nativeBuildInputs = [ gtk3 ];
+
+  src = fetchzip {
+    url = "https://github.com/EliverLara/candy-icons/archive/37b4566b4bfcdad99131a236ef1fc74db89a302e.zip";
```

Comment:
Why is this pulling a random commit? Also can we use `fetchFromGitHub` here?

Reply:
Unfortunately the maintainer of the package does not have some sort of release system on github nor tags for the versions.

Reply:
FYI: https://github.com/EliverLara/candy-icons/issues/59

Reply:
I'm not entirely sure about this, but I think it would okay to mark it with an unstable version since we're fetching from github. The version number only seems to be mentioned on https://www.opendesktop.org/p/1305251/

Reply:
It's been 3 weeks since I asked the maintainer of candy-icons for some sort of version system. Sadly didn't receive a response. So how should we proceed? Just mark it as unstable?

Reply:
Yes, I think it should be marked as unstable.